task_1/index.js

```diff
@@ -1 +1,11 @@
-console.log("true");
+const onMenuItemClick = (elementId) => {
+  const itemId = document.getElementById(elementId);
+  let items = document.getElementsByClassName("menu-item");
+  const active = "active";
+  for (var i = 0; i < items.length; i++) {
+    if (items[i].className.includes(active) === true) {
```

Comment:
There is no need to mention === true _here_ 
```suggestion
    if (items[i].className.includes(active)) {
```